Spikes/movement/Pointer.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#ifndef POINTER_H
+#define POINTER_H
+
+#include "SFML/Graphics.hpp" 
+#include "SFML/OpenGL.hpp" 
+
+using namespace sf;
+
+
+
+
+class Pointer
+{
+private:
+	Vector2f m_centre;
+	Vector2f m_pos;
+	float m_radius;
+	float m_angle;
+	float m_speed;
+	float m_minSpeed;
+	float m_maxSpeed;
+	float m_interval;
```

Comment:
no harm commenting some of these member variables that sound a little vague i.e. m_interval, time interval, angle interval? I don't want to read through functions to understand why a variable exists.